doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -356,12 +285,12 @@ Linear Algebra Equivalents
 
    * - ``(a>0.5)``
      - ``(a>0.5)``
-     - matrix whose i,jth element is (a_ij > 0.5).  The Matlab result is an
+     - matrix whose i,jth element is (a_ij > 0.5).  The MATLAB result is an
        array of 0s and 1s.  The NumPy result is an array of the boolean
        values ``False`` and ``True``.
 
    * - ``find(a>0.5)``
-     - ``nonzero(a>0.5)``
+     - ``where(a>0.5)``
```

Comment:
```suggestion
     - ``nonzero(a>0.5)``
```
The overloading of `where` just confuses users. Furthermore, the docs for `where` advise preferring `nonzero` anyway.